src/java.base/share/classes/java/util/concurrent/ForkJoinTask.java

```diff
@@ -1072,7 +1072,7 @@ public final void quietlyInvoke() {
 
     /**
      * Tries to join this task, returning true if it completed
-     * (possibly exceptionally) before the given timeout and
+     * (possibly exceptionally) before the given timeout elapses and
```

Comment:
Seems like mixed tense now: do we want completes/elapses or completed/elapsed?

Reply:
I admit I had doubts about the sequence of tenses too. That's a pretty complex sentence, and I'm a non-native English speaker. It's hard to re-tense it while retaining its conditional nature.

Reply:
Use "elapsed" here to match "completed".

Reply:
No, elsewhere you don't have an existing tense to match - I only want the sentence to be self-consistent. The question of whether the docs overall should be expressed in past or future tense is a much bigger issue.

Reply:
Oops I see the others do also have "completed" - my bad.